.github/workflows/fossa.yml

```diff
@@ -0,0 +1,20 @@
+name: fossa
+
+on:
+  push: {}
+  pull_request: {}
+
+jobs:
+  fossa-scan:
+    runs-on: ["org--${{ github.repository_owner }}--amd64--containers"]
+    container: ubuntu-22.04
+    steps:
+      - name : "Read vault secrets"
+        uses : rancher-eio/read-vault-secrets@main
+        with:
+          secrets: |
+            secret/data/github/repo/${{ github.repository }}/fossa/credentials api_key | FOSSA_API_KEY ;
+      - uses: actions/checkout@v3
+      - uses: fossas/fossa-action@main
```

Comment:
As per the migration document, it's preferred to use the commit hash instead of `vX`. For example:

```yaml
    - name: Checkout code
      # https://github.com/actions/checkout/releases/tag/v4.1.1
      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
    - name: Install Go
      # https://github.com/actions/setup-go/releases/tag/v5.0.0
      uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
```